### PR TITLE
Return icons in the API (fix #2639)

### LINKED ISSUE #2639: Return icons in the API
The API needs to return add-on icons URLs, with cachebusting string included. #2447 will need this as well.

### PR DESCRIPTION
**Pull request by diox**

### CODE REVIEW

File: src/olympia/addons/serializers.py [hunk 1]
```diff
@@ -152,8 +156,16 @@ def fake_object(self, data):
...
             ('average_daily_users', 'bayesian_rating', 'created',
-             'default_locale', 'guid', 'hotness', 'is_listed', 'last_updated',
-             'public_stats', 'slug', 'status', 'type', 'weekly_downloads'))
+             'default_locale', 'guid', 'hotness', 'icon_type', 'is_listed',
+             'last_updated', 'modified', 'public_stats', 'slug', 'status',
+             'type', 'weekly_downloads'))
+
+        # Temporary hack to make sure all add-ons have a modified date when
+        # serializing, to avoid errors when calling get_icon_url().
+        # Remove once all add-ons have been reindexed at least once since the
```
Comment (EnTeQuAk) [inline src/olympia/addons/serializers.py]: Let's mark this as "FIXME" and open an issue to track it?

**Comment by EnTeQuAk** [comment]
r+

File: src/olympia/addons/serializers.py (hunk 1 above)
Comment (diox) [inline src/olympia/addons/serializers.py]: I felt it was unnecessary since I'm spending most of my time looking at this file these days :)

Comment (EnTeQuAk) [inline src/olympia/addons/serializers.py]: fair enough :)

**Pull request merged**